lib/screen/account/recovery_phrase_page.dart

```diff
@@ -134,6 +158,162 @@ class RecoveryPhrasePage extends StatelessWidget {
+                  //'. You may also back it up to use it in another BIP-39 standard wallet:',
+                ),
+              ],
+            ),
+          ),
+          const SizedBox(height: 30),
+        ],
+      );
+    } else {
+      return Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          RichText(
+            text: TextSpan(
+              style: theme.textTheme.ppMori400Black14,
+              children: <TextSpan>[
+                Platform.isIOS
+                    ? TextSpan(
+                        text: 'icloud_keychain'.tr(),
+                        style: customLinkStyle,
+                        recognizer: TapGestureRecognizer()
+                          ..onTap = () => Navigator.of(context).pushNamed(
+                                  AppRouter.githubDocPage,
+                                  arguments: {
+                                    "prefix":
+                                        "/bitmark-inc/autonomy.io/main/apps/docs/",
+                                    "document": "security-ios.md",
+                                    "title": ""
+                                  }),
+                      )
+                    : TextSpan(
+                        text: 'android_block_store'.tr(),
+                        style: customLinkStyle,
+                        recognizer: TapGestureRecognizer()
+                          ..onTap = () => Navigator.of(context).pushNamed(
+                                  AppRouter.githubDocPage,
+                                  arguments: {
+                                    "prefix":
+                                        "/bitmark-inc/autonomy.io/main/apps/docs/",
+                                    "document": "security-android.md",
+                                    "title": ""
+                                  }),
+                      ),
+                TextSpan(
+                  text: '_is_'.tr(),
+                ),
+                TextSpan(
+                  text: 'turned_off'.tr().toLowerCase(),
+                  style: theme.textTheme.ppMori700Black14,
+                ),
+                TextSpan(
+                  text: 'unable_backup'.tr(),
+                  style: theme.textTheme.ppMori400Black14,
+                ),
+              ],
+            ),
+          ),
+          const SizedBox(height: 40),
+        ],
+      );
+    }
+  }
+
+  Widget _recommend() {
+    final theme = Theme.of(context);
+    if (_isBackUpAvailable != false) {
+      return const SizedBox();
+    }
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        const SizedBox(height: 30),
+        Text(
+            Platform.isAndroid
+                ? "recommend_google_cloud".tr()
+                : "recommend_icloud_key".tr(),
+            style: theme.textTheme.ppMori700Black14),
+      ],
+    );
+  }
+
+  Widget _settingButton() {
```

Comment:
pass and use the `context` in the Widget builder to avoid caching the old context.